### ISSUE TITLE
`EBADENGINE` warning shows up when installing the find-remove package

### ISSUE THREAD

**Issue by AnnAngela** [binarykitchen/find-remove#32]
When I installed the package, there is a warning message:
```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'find-remove@4.0.2',
npm WARN EBADENGINE   required: { node: '^16.1.0', npm: '^7.1.0' },
npm WARN EBADENGINE   current: { node: 'v16.13.0', npm: '8.1.3' }
npm WARN EBADENGINE }
```
I think that the npm version required can be writed as `>=7.1.0`, but I am not sure. Please help.

**Comment by binarykitchen** [comment]
Agreed. Sorry about the bad configuration. Fixed and published new v4.0.3.

**Comment by AnnAngela** [comment]
THX FOR FIX!
